collector/internal/telemetryapi/types.go

```diff
@@ -87,7 +87,7 @@ type SubscribeRequest struct {
 }
 
 type Event struct {
-	Time   string         `json:"time"`
-	Type   string         `json:"type"`
-	Record map[string]any `json:"record"`
+	Time   string `json:"time"`
+	Type   string `json:"type"`
+	Record any    `json:"record"`
```

Comment:
It looks like we change the type because it can be something other than a map?

Reply:
Yes.